### PR TITLE
Fix bug and windows compilation.

### PR DESCRIPTION
**Pull request by teams2ua**

### CODE REVIEW

File: core/src/wallet/bitcoin/transaction_builders/BitcoinLikeStrategyUtxoPicker.cpp [hunk 1]
```diff
@@ -438,7 +438,7 @@ namespace ledger {
             std::vector<size_t> indexes;
             auto const seed = std::chrono::system_clock::now().time_since_epoch().count();
 
-            indexes.reserve(utxos.size());
+            indexes.resize(utxos.size());
```
Comment (alekece) [inline core/src/wallet/bitcoin/transaction_builders/BitcoinLikeStrategyUtxoPicker.cpp:441]: Nice catch !

File: core/src/wallet/bitcoin/BitcoinLikeAccount.cpp [hunk 2]
```diff
@@ -630,7 +630,7 @@ namespace ledger {
         std::shared_ptr<api::BitcoinLikeTransactionBuilder> BitcoinLikeAccount::buildTransaction(std::experimental::optional<bool> partial) {
             auto self = std::dynamic_pointer_cast<BitcoinLikeAccount>(shared_from_this());
             auto getUTXO = [=]() -> Future<std::vector<BitcoinLikeUtxo>> {
-                return async<std::vector<BitcoinLikeUtxo>>([=]() {
+                return Future<std::vector<BitcoinLikeUtxo>>::async(_executionContext, [=]() {
```
Comment (alekece) [inline core/src/wallet/bitcoin/BitcoinLikeAccount.cpp:633]: Can you explain me what I was doing wrong here ?

Comment (teams2ua) [reply core/src/wallet/bitcoin/BitcoinLikeAccount.cpp:633]: Nothing wrong, just MSVC can't resolve between `async(...)` and `async(...) const` versions of the function.

Comment (alekece) [reply core/src/wallet/bitcoin/BitcoinLikeAccount.cpp:633]: Ok ! What about the `_executionContext` you are using in your change ?

Comment (teams2ua) [reply core/src/wallet/bitcoin/BitcoinLikeAccount.cpp:633]: I just "inline" `async` call. `BitcoinLikeAccount` is derived from `DedicatedContext` that contains `async` method and `_executionContext` member, so no semantic changes here.

**Pull request closed without merging**